<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindoww3</class>
 <widget class="QMainWindow" name="MainWindoww3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1711</width>
    <height>877</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="iconSize">
   <size>
    <width>24</width>
    <height>24</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="idEdit">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>380</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="nomEdit">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>440</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="descriptionEdit">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>500</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="prixEdit">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>560</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="ajouterButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>740</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(173, 216, 230); /* Bleu clair */
    color: rgb(255, 255, 255);  /* Texte blanc */
    border: 2px solid rgb(70, 130, 180); /* Bordure bleu foncé */
    border-radius: 20px; 
    padding: 10px; 
    transition: background-color 0.3s, color 0.3s, border 0.3s;
}

QPushButton:hover {
    background-color: rgb(240, 248, 255); /* Bleu très clair (presque blanc) pour le survol */
    color: black;  /* Texte noir lors du survol */
    border: 4px solid rgb(70, 130, 180); /* Bordure bleu foncé sur survol */
    outline: none; 
}</string>
    </property>
    <property name="text">
     <string>AJOUTER</string>
    </property>
   </widget>
   <widget class="QPushButton" name="supprimerButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>740</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(173, 216, 230); /* Bleu clair */
    color: rgb(255, 255, 255);  /* Texte blanc */
    border: 2px solid rgb(70, 130, 180); /* Bordure bleu foncé */
    border-radius: 20px; 
    padding: 10px; 
    transition: background-color 0.3s, color 0.3s, border 0.3s;
}

QPushButton:hover {
    background-color: rgb(240, 248, 255); /* Bleu très clair (presque blanc) pour le survol */
    color: black;  /* Texte noir lors du survol */
    border: 4px solid rgb(70, 130, 180); /* Bordure bleu foncé sur survol */
    outline: none; 
}</string>
    </property>
    <property name="text">
     <string>SUPPRIMER</string>
    </property>
   </widget>
   <widget class="QPushButton" name="modifierButton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>740</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(173, 216, 230); /* Bleu clair */
    color: rgb(255, 255, 255);  /* Texte blanc */
    border: 2px solid rgb(70, 130, 180); /* Bordure bleu foncé */
    border-radius: 20px; 
    padding: 10px; 
    transition: background-color 0.3s, color 0.3s, border 0.3s;
}

QPushButton:hover {
    background-color: rgb(240, 248, 255); /* Bleu très clair (presque blanc) pour le survol */
    color: black;  /* Texte noir lors du survol */
    border: 4px solid rgb(70, 130, 180); /* Bordure bleu foncé sur survol */
    outline: none; 
}</string>
    </property>
    <property name="text">
     <string>MODIFIER</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_ID_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>380</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>ID</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_ID_3">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>440</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Nom</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_ID_4">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>500</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Description</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_ID_5">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>560</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Prix</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_ID_6">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>620</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Categorie</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rechercheButton">
    <property name="geometry">
     <rect>
      <x>938</x>
      <y>20</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(173, 216, 230); /* Bleu clair */
    color: rgb(255, 255, 255);  /* Texte blanc */
    border: 2px solid rgb(70, 130, 180); /* Bordure bleu foncé */
    border-radius: 20px; 
    padding: 10px; 
    transition: background-color 0.3s, color 0.3s, border 0.3s;
}

QPushButton:hover {
    background-color: rgb(240, 248, 255); /* Bleu très clair (presque blanc) pour le survol */
    color: black;  /* Texte noir lors du survol */
    border: 4px solid rgb(70, 130, 180); /* Bordure bleu foncé sur survol */
    outline: none; 
}</string>
    </property>
    <property name="text">
     <string>RECHERCHE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="trierButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>740</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(173, 216, 230); /* Bleu clair */
    color: rgb(255, 255, 255);  /* Texte blanc */
    border: 2px solid rgb(70, 130, 180); /* Bordure bleu foncé */
    border-radius: 20px; 
    padding: 10px; 
    transition: background-color 0.3s, color 0.3s, border 0.3s;
}

QPushButton:hover {
    background-color: rgb(240, 248, 255); /* Bleu très clair (presque blanc) pour le survol */
    color: black;  /* Texte noir lors du survol */
    border: 4px solid rgb(70, 130, 180); /* Bordure bleu foncé sur survol */
    outline: none; 
}</string>
    </property>
    <property name="text">
     <string>TRIER</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exporterButton">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>740</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(173, 216, 230); /* Bleu clair */
    color: rgb(255, 255, 255);  /* Texte blanc */
    border: 2px solid rgb(70, 130, 180); /* Bordure bleu foncé */
    border-radius: 20px; 
    padding: 10px; 
    transition: background-color 0.3s, color 0.3s, border 0.3s;
}

QPushButton:hover {
    background-color: rgb(240, 248, 255); /* Bleu très clair (presque blanc) pour le survol */
    color: black;  /* Texte noir lors du survol */
    border: 4px solid rgb(70, 130, 180); /* Bordure bleu foncé sur survol */
    outline: none; 
}</string>
    </property>
    <property name="text">
     <string>EXPORTER</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="rechercheEdit">
    <property name="geometry">
     <rect>
      <x>348</x>
      <y>20</y>
      <width>581</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>rechercher...</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>10</y>
      <width>16</width>
      <height>791</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_employe">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(173, 216, 230); /* Bleu clair */
    color: rgb(255, 255, 255);  /* Texte blanc */
    border: 2px solid rgb(70, 130, 180); /* Bordure bleu foncé */
    border-radius: 20px; 
    padding: 10px; 
    transition: background-color 0.3s, color 0.3s, border 0.3s;
}

QPushButton:hover {
    background-color: rgb(240, 248, 255); /* Bleu très clair (presque blanc) pour le survol */
    color: black;  /* Texte noir lors du survol */
    border: 4px solid rgb(70, 130, 180); /* Bordure bleu foncé sur survol */
    outline: none; 
}</string>
    </property>
    <property name="text">
     <string>EMPLOYE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_navigate">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(173, 216, 230); /* Bleu clair */
    color: rgb(255, 255, 255);  /* Texte blanc */
    border: 2px solid rgb(70, 130, 180); /* Bordure bleu foncé */
    border-radius: 20px; 
    padding: 10px; 
    transition: background-color 0.3s, color 0.3s, border 0.3s;
}

QPushButton:hover {
    background-color: rgb(240, 248, 255); /* Bleu très clair (presque blanc) pour le survol */
    color: black;  /* Texte noir lors du survol */
    border: 4px solid rgb(70, 130, 180); /* Bordure bleu foncé sur survol */
    outline: none; 
}</string>
    </property>
    <property name="text">
     <string>PRODUIT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="serviceButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(173, 216, 230); /* Bleu clair */
    color: rgb(255, 255, 255);  /* Texte blanc */
    border: 2px solid rgb(70, 130, 180); /* Bordure bleu foncé */
    border-radius: 20px; 
    padding: 10px; 
    transition: background-color 0.3s, color 0.3s, border 0.3s;
}

QPushButton:hover {
    background-color: rgb(240, 248, 255); /* Bleu très clair (presque blanc) pour le survol */
    color: black;  /* Texte noir lors du survol */
    border: 4px solid rgb(70, 130, 180); /* Bordure bleu foncé sur survol */
    outline: none; 
}</string>
    </property>
    <property name="text">
     <string>SERVICE</string>
    </property>
   </widget>
   <widget class="QComboBox" name="categorieComboBox">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>620</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>nettoyage industriel </string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>nettoyage domestique</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>desinfection</string>
     </property>
    </item>
   </widget>
   <widget class="QTableView" name="serviceTableView">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>90</y>
      <width>661</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>40</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Saisir la remise (%)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="remiseLineEdit">
    <property name="geometry">
     <rect>
      <x>1240</x>
      <y>30</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>100</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Prix après remise :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="prixApresRemiseLabel">
    <property name="geometry">
     <rect>
      <x>1240</x>
      <y>90</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="remiseButton">
    <property name="geometry">
     <rect>
      <x>1490</x>
      <y>50</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(173, 216, 230); /* Bleu clair */
    color: rgb(255, 255, 255);  /* Texte blanc */
    border: 2px solid rgb(70, 130, 180); /* Bordure bleu foncé */
    border-radius: 20px; 
    padding: 10px; 
    transition: background-color 0.3s, color 0.3s, border 0.3s;
}

QPushButton:hover {
    background-color: rgb(240, 248, 255); /* Bleu très clair (presque blanc) pour le survol */
    color: black;  /* Texte noir lors du survol */
    border: 4px solid rgb(70, 130, 180); /* Bordure bleu foncé sur survol */
    outline: none; 
}</string>
    </property>
    <property name="text">
     <string>REMISE</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>210</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Prix converti :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="deviseComboBox">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>200</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>USD</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>EUR</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>GBP</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>JPY</string>
     </property>
    </item>
   </widget>
   <widget class="QPlainTextEdit" name="prixConvertiLabel">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>240</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="convertirButton">
    <property name="geometry">
     <rect>
      <x>1370</x>
      <y>210</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(173, 216, 230); /* Bleu clair */
    color: rgb(255, 255, 255);  /* Texte blanc */
    border: 2px solid rgb(70, 130, 180); /* Bordure bleu foncé */
    border-radius: 20px; 
    padding: 10px; 
    transition: background-color 0.3s, color 0.3s, border 0.3s;
}

QPushButton:hover {
    background-color: rgb(240, 248, 255); /* Bleu très clair (presque blanc) pour le survol */
    color: black;  /* Texte noir lors du survol */
    border: 4px solid rgb(70, 130, 180); /* Bordure bleu foncé sur survol */
    outline: none; 
}</string>
    </property>
    <property name="text">
     <string>CONVERTIR</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="statistiquesWidget">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>310</y>
      <width>351</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="statistiquesButton">
    <property name="geometry">
     <rect>
      <x>1480</x>
      <y>360</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(173, 216, 230); /* Bleu clair */
    color: rgb(255, 255, 255);  /* Texte blanc */
    border: 2px solid rgb(70, 130, 180); /* Bordure bleu foncé */
    border-radius: 20px; 
    padding: 10px; 
    transition: background-color 0.3s, color 0.3s, border 0.3s;
}

QPushButton:hover {
    background-color: rgb(240, 248, 255); /* Bleu très clair (presque blanc) pour le survol */
    color: black;  /* Texte noir lors du survol */
    border: 4px solid rgb(70, 130, 180); /* Bordure bleu foncé sur survol */
    outline: none; 
}</string>
    </property>
    <property name="text">
     <string>STATISTIQUES</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-40</y>
      <width>1831</width>
      <height>911</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../462551453_564312012889256_303978322915016033_n.png</pixmap>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="serviceDateTime">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>670</y>
      <width>194</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));
font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="scheduleServiceButton">
    <property name="geometry">
     <rect>
      <x>1340</x>
      <y>690</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(173, 216, 230); /* Bleu clair */
    color: rgb(255, 255, 255);  /* Texte blanc */
    border: 2px solid rgb(70, 130, 180); /* Bordure bleu foncé */
    border-radius: 20px; 
    padding: 10px; 
    transition: background-color 0.3s, color 0.3s, border 0.3s;
}

QPushButton:hover {
    background-color: rgb(240, 248, 255); /* Bleu très clair (presque blanc) pour le survol */
    color: black;  /* Texte noir lors du survol */
    border: 4px solid rgb(70, 130, 180); /* Bordure bleu foncé sur survol */
    outline: none; 
}</string>
    </property>
    <property name="text">
     <string>Planifier</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="confirmationMessage">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>730</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_commande">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>530</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(173, 216, 230); /* Bleu clair */
    color: rgb(255, 255, 255);  /* Texte blanc */
    border: 2px solid rgb(70, 130, 180); /* Bordure bleu foncé */
    border-radius: 20px; 
    padding: 10px; 
    transition: background-color 0.3s, color 0.3s, border 0.3s;
}

QPushButton:hover {
    background-color: rgb(240, 248, 255); /* Bleu très clair (presque blanc) pour le survol */
    color: black;  /* Texte noir lors du survol */
    border: 4px solid rgb(70, 130, 180); /* Bordure bleu foncé sur survol */
    outline: none; 
}</string>
    </property>
    <property name="text">
     <string>COMMANDE</string>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>idEdit</zorder>
   <zorder>nomEdit</zorder>
   <zorder>descriptionEdit</zorder>
   <zorder>prixEdit</zorder>
   <zorder>ajouterButton</zorder>
   <zorder>supprimerButton</zorder>
   <zorder>modifierButton</zorder>
   <zorder>lineEdit_ID_2</zorder>
   <zorder>lineEdit_ID_3</zorder>
   <zorder>lineEdit_ID_4</zorder>
   <zorder>lineEdit_ID_5</zorder>
   <zorder>lineEdit_ID_6</zorder>
   <zorder>rechercheButton</zorder>
   <zorder>trierButton</zorder>
   <zorder>exporterButton</zorder>
   <zorder>rechercheEdit</zorder>
   <zorder>line</zorder>
   <zorder>pushButton_employe</zorder>
   <zorder>pushButton_navigate</zorder>
   <zorder>serviceButton</zorder>
   <zorder>categorieComboBox</zorder>
   <zorder>serviceTableView</zorder>
   <zorder>label_7</zorder>
   <zorder>remiseLineEdit</zorder>
   <zorder>label_8</zorder>
   <zorder>prixApresRemiseLabel</zorder>
   <zorder>remiseButton</zorder>
   <zorder>label_10</zorder>
   <zorder>deviseComboBox</zorder>
   <zorder>prixConvertiLabel</zorder>
   <zorder>convertirButton</zorder>
   <zorder>statistiquesWidget</zorder>
   <zorder>statistiquesButton</zorder>
   <zorder>serviceDateTime</zorder>
   <zorder>scheduleServiceButton</zorder>
   <zorder>confirmationMessage</zorder>
   <zorder>pb_commande</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1711</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
